### ISSUE TITLE
Interaction of fixed auxiliary parameters & extract_draws/parse_bf in nonlinear models

### ISSUE THREAD

**Issue by aforren1** [paul-buerkner/brms#171]
This came up when I was trying out the new quantile regression (most recent master). Example with the built-in orange tree growth data:
```r
m1 <- brm(bf(circumference ~ asym/(1 + exp((xmid - age)/scal)),
              asym ~ 1, xmid ~ 1, scal ~ 1, nl = TRUE,
              quantile = 0.5),
           data = Orange, 
           prior = c(prior('normal(200, 10)', nlpar = 'asym'),
                     prior('normal(750, 10)', nlpar = 'xmid'),
                     prior('normal(350, 10)', nlpar = 'scal')),
           family = asym_laplace(),
           chains = 1)

# call anything that calls extract_draws/parse_bf
predict(m2)
##Error: Invalid auxiliary parameters: 'quantile'
```

I *think* (not sure) the fix is to NULL pfix in `.extract_draws` around line 126, e.g.
```r
...
  x$formula$pforms <- NULL
  x$formula$pfix <- NULL
  x$formula$nl <- FALSE
...
```
which seems to work for me (on a fork, I can make a PR if it's easier).

**Comment by paul-buerkner** [comment]
Thanks for reporting this problem! I will push a commit in a few minutes fixing it! :-)

**Comment by paul-buerkner** [comment]
Fixed via 23db49e. Indeed, the fix was exactly as you said, thanks again!
